src/DataObjects/QueuedJobDescriptor.php

```diff
@@ -585,4 +611,17 @@ private function getWorkerExpiry()
 
         return $time->getTimestamp() - $now->getTimestamp();
     }
+
+    public function getDefaultSearchContext()
```

Comment:
Because JobStatus and JobType are VARCHARs rather than ENUM, they will default to Text field instead of dropdowns.

This bit of logic force them to use drop down.